assets/thanos-querier/deployment.yaml

```diff
@@ -76,6 +76,15 @@ spec:
           requests:
             cpu: 10m
             memory: 12Mi
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
```

Comment:
I think the containers below can benefit from this as well.
- `prometheus-operator-admission-webhook`
- `kube-state-metrics`
- `kube-rbac-proxy-{main,self}`
- `init-textfile`

Reply:
maybe in a follow up PR?

Reply:
@rexagod Can you file a JIRA ticket for this please? Might even be worth a bug?